src/language-css/printer-postcss.js

```diff
@@ -115,12 +115,20 @@ function genericPrint(path, options, print) {
         options.locStart(node),
         options.locEnd(node)
       );
+
       const rawText = node.raws.text || node.text;
       // Workaround a bug where the location is off.
       // https://github.com/postcss/postcss-scss/issues/63
       if (text.indexOf(rawText) === -1) {
-        if (node.raws.inline) {
-          return concat(["// ", rawText]);
+        if (node.raws.inline || node.inline) {
+          const needBreakAfter = !(
+            node.source.input.css.split("\n")[node.source.end.line] || ""
+          ).trim();
+          return concat([
+            "//",
+            node.raws.left + rawText,
+            needBreakAfter ? "\n" : ""
+          ]);
```

Comment:
"\n"? We should use prettier builders
`node.source.input.css` should be never used

this place is very bad, need rewrite this